chia/server/ws_connection.py

```diff
@@ -118,6 +124,7 @@ def __init__(
         self.protocol_version = ""
 
     def _get_extra_info(self, name: str) -> Optional[Any]:
+        assert self.ws._writer is not None, "ws._writer is None"
```

Comment:
What attribute error? I have no idea when this would trigger but it didn't fail anywhere so far i guess otherwise we would have seen reports about it.

Reply:
Fun, I just realized that we're accessing a private attribute of a library's class.  Sounds like time for an upstream issue or PR to discuss how to make this better either with changes here or changes to aiohttp.

The line below does, in part, `self.ws._writer.transport` which would fail with an `AttributeError` in case of `self.ws._writer` being `None`.  I understand that this assertion is being added for complete handling of the hinted possibilities, but having a more useful error while we're at it seems good.  It may be that with present code it is literally impossible to trigger this assertion.  So, what bug would be likely to be written that would make it possible?

I don't know, I'm starting to feel partially silly about pushing on this.  But, I feel like _something_ is off when we have random assertions getting tossed around "to placate the mypy gods", as some might say, and so trying to look around a bit more and add a bit more value seems good.  Maybe it's just the private attribute access here.

```suggestion
        assert self.ws._writer is not None, "websocket's ._writer is None, was .prepare() called?"
```

Reply:
Initially i wanted to just do #13869 but this only works for the `ClientWebSocketResponse`, not for `WebSocketResponse`, it misses this wrapper.

Reply:
Sounds like `WebSocketResponse` is missing the method?